docs/source/getting-started.rst

```diff
@@ -385,6 +385,7 @@ The resulted item will look like this:
 
 .. _`JSON Schema`: https://json-schema.org/
 .. _`schematics`: https://schematics.readthedocs.io/en/latest/
+.. _`cerberus`: https://docs.python-cerberus.org/en/latest/index.html
```

Comment:
:lipstick: Please, keep the alphabetical order :slightly_smiling_face:

Reply:
Yes, good catch. I will rectify this in other places as well. 
Thanks @Gallaecio

Reply:
Fixed other files as well, just for good 'ol uniformity.